### ISSUE TITLE
DatePickerIOS is not accessible

### ISSUE THREAD

**Issue by oargaruna** [facebook/react-native#13486]
### Description

DatePickerIOS does not receive VoiceOver focus by swiping left or right on the screen.

### Reproduction

1. Add a Touchable to any React Native View which will launch the DatePickerIOS when tapped.
2. Activate VoiceOver on an iPhone.
3. Bring Talkback focus to the Touchable button. Double tap to open the date picker.
4. Swiping left-right does not bring accessibility focus to any of the controls in DatePickerIOS.

### Additional Information

* React Native version: 0.41
* Platform: iOS

**Comment by hramos** [comment]
Hi there! This issue is being closed because {:search=>"repo:facebook/react-native is:open is:issue updated:<2017-05-25 -label:\"Good First Task\" -label:\"Help Wanted\" ", :message=>"it has been inactive for a while"}. Maybe the issue has been fixed in a recent release, or perhaps it is not affecting a lot of people. Either way, we're automatically closing issues after a period of inactivity. Please do not take it personally!

If you think this issue should definitely remain open, please let us know. The following information is helpful when it comes to determining if the issue should be re-opened:

- Does the issue still reproduce on the latest release candidate? Post a comment with the version you tested.
- If so, is there any information missing from the bug report? Post a comment with all the information required by the [issue template](https://github.com/facebook/react-native/blob/master/.github/ISSUE_TEMPLATE.md).
- Is there a pull request that addresses this issue? Post a comment with the PR number so we can follow up.

If you would like to work on a patch to fix the issue, *contributions are very welcome*! Read through the [contribution guide](http://facebook.github.io/react-native/docs/contributing.html), and feel free to hop into [#react-native](https://discordapp.com/invite/0ZcbPKXt5bZjGY5n) if you need help planning your contribution.
